packages/cli/src/codegen/schema.ts

```diff
@@ -104,13 +105,31 @@ export default class SchemaCodeGenerator {
       .filter(Boolean) as Array<tsCodegen.Class>;
   }
 
+  generateDerivedLoaders() {
+    const fields = this.schema.ast.definitions
+      .filter(def => this._isEntityTypeDefinition(def))
+      .flatMap((def: any) => def.fields)
+      .filter((def: any) => this._isDerivedField(def))
+      .map((def: FieldDefinitionNode) => this._getTypeNameForField(def.type));
```

Comment:
let's not use `any` type here. When you filter ` .filter(def => this._isEntityTypeDefinition(def))` then it returns a `ObjectTypeDefinitionNode`  which is more narrowed down type

Reply:
understood, but removing that any resulted in some weird errors, i have now manually casted these arrays to `ObjectTypeDefinitionNode[]` that seemed to have solved the problem